server/datastore/mysql/hosts_test.go

```diff
@@ -6558,6 +6558,12 @@ func testHostsDeleteHosts(t *testing.T, ds *Datastore) {
 	`, host.UUID)
 	require.NoError(t, err)
 
+	_, err = ds.writer(context.Background()).Exec(`
+          INSERT INTO host_mdm_apple_declarations (host_uuid, declaration_uuid)
+          VALUES (?, uuid())
```

Comment:
Should we add the "d" prefix here or did you want to leave it out for test purposes?

Reply:
exactly, I figured it wasn't relevant. Can add it!